homeassistant/components/flux_led/light.py

```diff
@@ -330,7 +338,7 @@ def update(self):
                     self._error_reported = True
                 return
         self._bulb.update_state(retry=2)
-        if self._mode != MODE_WHITE and self._bulb.getRgb() != (0, 0, 0):
+        if self._mode != MODE_WHITE:
```

Comment:
In warm-white mode `self._bulb.getRgb()` will always be `(0,0,0)`  while `self._bulb.getRgbw()` will return `(0,0,0,0-255)` value. That's why the condition `and self._bulb.getRgb() != (0, 0, 0)` needs to be removed if we want to be able to work in both modes RGB and "warm-white"